app/views/how-tos/index.html

```diff
@@ -91,15 +91,37 @@ <h2 class="nhsuk-heading-s">General use</h2>
         <li><a href="/how-tos/components">Using components</a></li>
         <li><a href="/how-tos/passing-data-page">Passing data page to page</a></li>
         <li><a href="/how-tos/branching">Branching</a> – show a different page based on user input</li>
+
       </ul>
 
-        <h2 class="nhsuk-heading-s">Share your prototype</h2>
+        <h2 class="nhsuk-heading-s">Storing and sharing code</h2>
 
       <ul class="nhsuk-list">
-        <li><a href="/how-tos/git">Setting up Git</a></li>
+        <li><a href="/how-tos/github/index">Store your code online with GitHub and GitHub Desktop</a>
```

Comment:
probably wants to link to `how-tos/github`

Reply:
changed (along with ordering as this is going to the old guidance for just git as a concept) - i'm leaving that page in for now to decide whether it's worth taking out or not